src/drakvuf.cpp

```diff
@@ -406,6 +406,12 @@ static bool is_terminated(drakvuf_t drakvuf, void* data)
             info->proc->at(info->pid));
 };
 
+/* FIXME This could break "procdump" plugin
```

Comment:
Please remove this comment and open an Issue instead so that it's properly tracked

Reply:
Done. #1289